hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/states/ContainerStateMap.java

```diff
@@ -85,7 +84,7 @@
 
```

Comment:
You may want to update the [documentation](https://github.com/apache/ozone/blob/e7e984c559dabe648f4ab81d0f7d901ffcc50fe6/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/states/ContainerStateMap.java#L65) of this class. 

Currently it's talking about factor.